lib/app.rb

```diff
@@ -0,0 +1,71 @@
+require_relative "./item_repository"
+require_relative "./order_repository"
+require_relative "./database_connection"
+
+class Application
+  def initialize(database_name, io, item_repository, order_repository)
+    DatabaseConnection.connect(database_name)
+    @io = io
+    @item_repository = item_repository
+    @order_repository = order_repository
+  end
+
```

Comment:
I thought I commented this before but it disappeared. The run method is very long, consider splitting up the functionality of the run method into separate methods for each choice the user makes.